migrations/env.py

```
@@ -0,0 +1,80 @@
+from logging.config import fileConfig
+
+from alembic import context
+from sqlalchemy import engine_from_config, pool
+from sqlalchemy.ext.asyncio import async_engine_from_config
+
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
+
+# Import your project's Base metadata here
+try:
+    from easy_access.db.models_base import metadata as target_metadata
+except Exception:  # pragma: no cover - keep imports tolerant during scaffolding
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)